backendapi/views.py

```diff
@@ -27,3 +29,42 @@ def api_get(request):
         return Response({"message": "GET request received"}, status=200)
     elif request.method == "POST":
         return Response({"message": "POST request received"}, status=200)
+
+
+@api_view(["GET"])
+def verify_ticket(request, ticket):
+    # TODO: Implement ticket check
+    if ticket == 'test123':
+        return Response(
+            {"is_valid": True}, status=status.HTTP_200_OK,
+        )
+    return Response(
+        {"is_valid": False}, status=status.HTTP_400_BAD_REQUEST,
+    )
+
+
+@api_view(["POST"])
+def create_attendant(request):
+    ticket_id = request.data.get("ticket_id")
+    nfc_id = b16decode(request.data.get("nfc_id"), casefold=True)
+
+    if not ticket_id or not nfc_id:
+        return Response(
+            {"error": "ticket_id and nfc_id are required."},
+            status=status.HTTP_400_BAD_REQUEST,
+        )
+
+    attendant = Attendant.objects.create(ticket_id=ticket_id, nfc_id=nfc_id)
+
+    return Response(
+        {
+            "message": "Attendant created successfully.",
+            "data": {
+                "ticket_id": attendant.ticket_id,
+                "nfc_id": b16encode(attendant.nfc_id),
+                "discord": "",
+                "is_crew": attendant.is_crew,
+                "is_valid": attendant.is_valid,
+            }
```

Comment:
```suggestion
            "id": attendant.id,
```

`id` is the only field that would be unknown to the client, and `discord`, `is_crew` and `is_valid` will never have anything but the default values.

We can add a return of the whole object if we find that we need it later, but then we should probably use a generator